tests/gui/models/selection-state.spec.js

```diff
@@ -403,20 +433,62 @@ describe('Browser: SelectionState', function() {
           }).to.throw('Missing image size');
         });
 
-        it('should throw if size is not a number', function() {
+        it('should throw if the original size is not a number', function() {
+          m.chai.expect(function() {
+            SelectionStateModel.setImage({
+              path: 'foo.img',
+              size: {
+                original: '999999999',
+                final: {
+                  estimation: false,
+                  value: 999999999
+                }
+              }
+            });
+          }).to.throw('Invalid original image size: 999999999');
```

Comment:
That doesn't look trivial to implement, unless we check if the invalid thing is a string, and then manually quote ourselves, otherwise we'd be unnecessarily quoting things like objects, or booleans. This errors are things being caught by our test suite, so I think its fine. Seeing that `999999999` is invalid instantly leads to the fact that is really a string after taking a glance at the code.